src/MCPClient/tests/test_clamdscan.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf8 -*-
+"""Tests for the archivematicaClamscan.py client script."""
+
+from __future__ import print_function
+
+import os
+import sys
+
+from collections import namedtuple
+from clamd import ClamdNetworkSocket, ClamdUnixSocket
+
+THIS_DIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.abspath(os.path.join(THIS_DIR, '../lib/clientScripts')))
+
+import archivematicaClamscan
+
+# ClamdScanner tests
```

Comment:
Same here, I think we should use the module docstring.

Reply:
Resolved.